docs/compute-systems/jupyter-and-ipython.md

```diff
@@ -16,21 +16,20 @@ language interpreters. See details below.
 
 The instructions below describe how to start the
 browser-based **JupyterLab**, the **IPython shell**, and **Jupyter
-QtConsole** on the NCAR systems.
-visualization nodes.
+QtConsole** on the NCAR HPC systems.
 
 For additional information, see [Jupyter documentation](https://jupyter.readthedocs.io/en/latest/).
 
 ## Starting JupyterLab
 
 - Start an interactive job using the `qinteractive` command.
   (Alternative: Start the job on Casper using `execcasper` if it
```

Comment:
I'd suggest swapping this language here and in the following two sections to list Casper first, since that is the preferred platform for this workflow at this point.

Reply:
Swapped. Also added language to discourage Derecho use given memory for each node.